src/samplers/krb5kdc/stat.rs

```diff
@@ -0,0 +1,706 @@
+    ProcessTgsReqServiceExp,
+
+    #[strum(serialize = "krb5kdc/process_tgs_req/bad_pvno")]
+    ProcessTgsReqBadPvno,
+
+    #[strum(serialize = "krb5kdc/process_tgs_req/c_old_mast_kvno")]
+    ProcessTgsReqCOldMastKvno,
+
+    #[strum(serialize = "krb5kdc/process_tgs_req/s_old_mast_kvno")]
+    ProcessTgsReqSOldMastKvno,
+
+    #[strum(serialize = "krb5kdc/process_tgs_req/c_principal_unknown")]
+    ProcessTgsReqCPrincipalUnknown,
+
+    #[strum(serialize = "krb5kdc/process_tgs_req/s_principal_unknown")]
+    ProcessTgsReqSPrincipalUnknown,
+
+    #[strum(serialize = "krb5kdc/process_tgs_req/principal_not_unique")]
+    ProcessTgsReqPrincipalNotUnique,
+
+    #[strum(serialize = "krb5kdc/process_tgs_req/null_key")]
+    ProcessTgsReqNullKey,
+
+    #[strum(serialize = "krb5kdc/process_tgs_req/cannot_postdate")]
+    ProcessTgsReqCannotPostdate,
+
+    #[strum(serialize = "krb5kdc/process_tgs_req/never_valid")]
+    ProcessTgsReqNeverValid,
+
+    #[strum(serialize = "krb5kdc/process_tgs_req/policy")]
+    ProcessTgsReqPolicy,
+
+    #[strum(serialize = "krb5kdc/process_tgs_req/badoption")]
+    ProcessTgsReqBadoption,
+
+    #[strum(serialize = "krb5kdc/process_tgs_req/etype_nosupp")]
+    ProcessTgsReqEtypeNosupp,
+
+    #[strum(serialize = "krb5kdc/process_tgs_req/sumtype_nosupp")]
+    ProcessTgsReqSumtypeNosupp,
+
+    #[strum(serialize = "krb5kdc/process_tgs_req/padata_type_nosupp")]
+    ProcessTgsReqPadataTypeNosupp,
+
+    #[strum(serialize = "krb5kdc/process_tgs_req/trtype_nosupp")]
+    ProcessTgsReqTrtypeNosupp,
+
+    #[strum(serialize = "krb5kdc/process_tgs_req/client_revoked")]
+    ProcessTgsReqClientRevoked,
+
+    #[strum(serialize = "krb5kdc/process_tgs_req/service_revoked")]
+    ProcessTgsReqServiceRevoked,
+
+    #[strum(serialize = "krb5kdc/process_tgs_req/tgt_revoked")]
+    ProcessTgsReqTgtRevoked,
+
+    #[strum(serialize = "krb5kdc/process_tgs_req/client_notyet")]
+    ProcessTgsReqClientNotyet,
+
+    #[strum(serialize = "krb5kdc/process_tgs_req/service_notyet")]
+    ProcessTgsReqServiceNotyet,
+
+    #[strum(serialize = "krb5kdc/process_tgs_req/key_exp")]
+    ProcessTgsReqKeyExp,
+
+    #[strum(serialize = "krb5kdc/process_tgs_req/preauth_failed")]
+    ProcessTgsReqPreauthFailed,
+
+    #[strum(serialize = "krb5kdc/process_tgs_req/preauth_required")]
+    ProcessTgsReqPreauthRequired,
+
+    #[strum(serialize = "krb5kdc/process_tgs_req/server_nomatch")]
+    ProcessTgsReqServerNomatch,
+
+    #[strum(serialize = "krb5kdc/process_tgs_req/must_use_user2user")]
+    ProcessTgsReqMustUseUser2user,
+
+    #[strum(serialize = "krb5kdc/process_tgs_req/path_not_accepted")]
+    ProcessTgsReqPathNotAccepted,
+
+    #[strum(serialize = "krb5kdc/process_tgs_req/svc_unavailable")]
+    ProcessTgsReqSvcUnavailable,
+}
+
+impl Krb5kdcStatistic {
+    pub fn bpf_table(self) -> &'static str {
+        match self {
+            Self::FinishProcessAsReqUnknown => "counts_finish_process_as_req",
+
```

Comment:
I'm not sure theres any readability benefit to these newlines in the match

Reply:
Cleaned up.